### DISCUSSION TITLE
Using the stop watch - and leave page for actual work

### DISCUSSION THREAD

**Discussion by degudejung** [kromitgmbh/titra#86] [General]
Hi,
first of all thanks for that nice piece of software! I set up titra in a docker in no time can it's working ootb.

One thing I maybe misunderstand or I would find super helpful is the following:

When hitting "+ track" I choose a project and enter a task description (or pick one from list). Then I hit the timer button to trigger the stop watch. Normally, I then leave the window or maybe even shutdown the browser and work on whatever it is. Then, after finishing the task I return to titra and enjoy to see that the timer is still counting even if the window was closed. So I click it. But now all fields are empty and I need to fill them in again. Is that how it's supposed to work? It would be great if the values could stick, once typed in.

Also, when I have settings on "hours", the timer seems to put in the right values. When I have setting on "minutes", it does not seem to convert from hours to minutes correctly, so 6 minutes on the timer would convert to 0.1 instead of 6.0 in the entry.

Lastly, but this is more of a feature request; it would be awesome to use the timer to capture the start date/time and stop date/time instead of the time difference. Why? On many projects, we need to protocol not only how much time we spent (2.5 hours) but also, when we spent that time (13:00-15:30, 2.5 hours).
UPDATE: OK, I saw the "log start time" can be activated in settings. However, it will not appear in the report, neither in the internal table nor in the exported files. Would that be something you could add easily?

Thanks and good luck!

**Comment by faburem** [comment]
Thanks for your feedback! We can definitely improve the timer feature and store more information along with it. I will add all your suggestions to our roadmap in the next couple of days, thanks again we really rely on input like this! 👍

**Comment by faburem** [comment]
This has just been released in titra version 0.63.0! Please check it out and let me know what you think.

Happy holidays 🎅🏻
